### PR TITLE
Update Vagrantfile

### PR DESCRIPTION
**Pull request by manics**
Update Vagrantfile.
Also bump playbook to PostgreSQL 10 (the molecule test already uses 10).

Companion to https://github.com/ome/training-ansible/pull/1

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
-    ansible.playbook = "playbook.yml"
-    ansible.galaxy_role_file = "../requirements.yml"
+  config.vm.provider "virtualbox" do |vb|
+    vb.memory = "2048"
+    vb.cpus = "2"
   end
+
+  # Uncomment this to automatically run Ansible
+  # config.vm.provision "ansible" do |ansible|
+  #   ansible.playbook = "playbook.yml"
```
Comment (joshmoore) [inline Vagrantfile:20]: Would you mind making this callable via the command-line? https://www.vagrantup.com/docs/provisioning/basic_usage.html#run-once-always-or-never

**Review by joshmoore — APPROVED** [review]
One minor comment. Otherwise, :+1:

File: Vagrantfile (hunk 1 above)
Comment (manics) [reply Vagrantfile:20]: I've tried
```
  # Run the Ansible provisioner after the VM has started:
  # vagrant provision --provision-with ansible
  config.vm.provision "ansible", run: "never" do |ansible|
    ansible.playbook = "playbook.yml"
    ansible.galaxy_role_file = "requirements.yml"
  end
```
but I can't get it to run, neither `vagrant provision --provision-with ansible` nor `vagrant provision` work

Comment (joshmoore) [reply Vagrantfile:20]: Alright. Probably not worth any more time investment. In general, I find "uncomment this block" unfortunate from a usability point-of-view, but better than nothing.

**Comment by joshmoore** [comment]
Note sure why this didn't get merged.

**Pull request merged**